docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/ecommerce/original/index.md

```diff
@@ -0,0 +1,394 @@
+---
+title: "Original"
+sidebar_position: 70
+---
+
+# Ecommerce tracking (original plugin)
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+This plugin has been superseded by the [Snowplow ecommerce plugin](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/ecommerce/index.md). We highly recommend using this newer plugin, which is more fully featured and allows you to use the DBT model we provide.
```

Comment:
Could you standardise this text with the other page and maybe make it an admon so it's a bit more obvious?